jiant/trainer.py

```diff
@@ -30,10 +29,12 @@
     assert_for_log,
     find_last_checkpoint_epoch,
     check_for_previous_checkpoints,
+    get_output_attribute,
+    get_model_attribute,
 )  # pylint: disable=import-error
 
 
-def build_trainer_params(args, task_names, phase="pretrain"):
+def build_trainer_params(args, use_cuda, cuda_device, task_names, phase="pretrain"):
```

Comment:
Can we merge these ? e.g. if `use_cuda = bool(cuda_device < 0)`

Reply:
The alternative is to have a function that checks whether to use_cuda based on the cuda_device.